Userland/DynamicLoader/main.cpp

```diff
@@ -103,6 +103,11 @@ static ErrorOr<int> open_executable(char const* path)
     return checked_fd;
 }
 
+[[gnu::always_inline]] static void optimizer_fence()
```

Comment:
Does `ALWAYS_INLINE` work here? we don't use the GNU-specific attribute anywhere else AFAICT

Reply:
I think `ALWAYS_INLINE` will just paper over the issue. If GCC says it might not inline `optimizer_fence`, `optimizer_fence` should be a macro.

Reply:
Is there a documented difference between how GCC implements `__attribute__((always_inline))` and `[[gnu::always_inline]]`?

Reply:
Oh, I always thought that ALWAYS_INLINE is `[[gnu::always_inline]]` + suppression of "might not be inlinable" warning (that's why I wrote the previous comment).

Not sure how `static` and `static inline` functions differ in semantics (might be a GCC bug) but seems like adding `inline` (i. e. using `ALWAYS_INLINE`) fixes the warning.